wp-content/themes/csisnuclearnetwork/assets/_scss/components/_entry-header.scss

```diff
@@ -119,14 +128,135 @@
     }
   }
 
-  &__title {
-    @extend %text-style-heading-3-x-large-strong;
-    color: $color-white-190;
+  &--light {
+    --page-header-title-color: #{$color-accent-primary-blue-100};
+    --page-header-title-margin-bottom: #{rem(16)};
+    /* stylelint-disable-next-line */
+    $headerCircles: '/wp-content/themes/csisnuclearnetwork/assets/static/page-header-circles.svg';
+    /* stylelint-disable-next-line */
+    $gradient: url($headerCircles), linear-gradient(180deg, rgba(16, 48, 79, 0.68) 0%, rgba(53, 110, 163, 0.0966914) 30.95%, rgba(56, 115, 171, 0) 100%), linear-gradient(90.1deg, #1c4973 0.76%, #08223a 100.12%);
+
+    max-width: $size__content-wide-max-width;
+    margin: 0 calc(-1 * var(--container-padding)) rem(24);
+    padding: calc(#{rem(32)} + var(--site-header-height)) var(--container-padding) rem(32); // Top padding includes 56px for the site header
+    background-color: $color-bg-light-100;
+
+    @include breakpoint('small') {
+      margin: 0 auto rem(40);
+    }
+
+    @include breakpoint('medium') {
+      padding-top: calc(#{rem(40)} + var(--site-header-height));
+    }
+
+    @include breakpoint('large') {
+      padding-top: calc(#{rem(64)} + var(--site-header-height));
+    }
+
+    &::before {
+      height: 100%;
+      background: $gradient;
+
+      @include breakpoint('small') {
+        @include csis-block-full-width-wrapper();
+        right: calc(-1 * calc((100vw - 100%) / 2));
```

Comment:
I'm not really sure what the `right` and `width` properties are meant to be doing here? Removing them doesn't change anything that I can see.